resources/PageApprovals.css

```diff
@@ -0,0 +1,258 @@
+.page-approval-container {
+	position: absolute;
+	top: -55px;
+	right: 0;
+	z-index: 1;
+	display: flex;
+	align-items: center;
+}
+
+.tooltip {
+	visibility: hidden;
+	position: absolute;
+}
+
+.approval-status-badge {
+	display: flex;
+	align-items: center;
+	padding: 4px 15px;
+	border-radius: 20px;
+	font-weight: bold;
+	font-size: 12px;
+	cursor: pointer;
+	position: relative;
+}
+
+.approved-badge {
+	color: #0f7958;
+	background-color: rgba(209, 249, 216, 0.75);
+	padding: 5px 30px;
+}
+
+.unapproved-badge {
+	background-color: lightgray;
+	color: black;
+	padding: 5px 23px;
+}
+
+.approved-badge:hover {
+	background-color: #bce0c2;
+}
+
+.status-icon, .checkmark-icon, .unapprove-icon {
+	background-size: contain;
+	background-repeat: no-repeat;
+	background-position: center;
+}
+
+.status-icon {
+	width: 22px;
+	height: 22px;
+	margin-right: 5px;
+}
+
+.approved-badge .status-icon {
+	background-image: url('Icons/approved-icon.svg');
+	margin-right: 10px;
+}
+
+.unapproved-badge .status-icon {
+	background-image: url('Icons/not-approved-icon.svg');
+}
+
+.checkmark-icon {
+	width: 24px;
+	height: 24px;
+	background-image: url('Icons/checkmark.svg');
+}
+
+#unapproveButton .unapprove-icon {
+	width: 20px;
+	height: 20px;
+	background-image: url('Icons/unapprove-icon.svg');
+}
+
+.approver-details {
+	font-size: 12px;
+}
+
+.page-approval-text-muted {
+	color: gray;
+	display: inline-block;
+	width: 110px;
+	text-align: center;
+}
+
+.page-approval-text-muted::first-letter {
+	text-transform: uppercase;
+}
+
+.approved-status-text {
+	user-select: none;
+}
+
+.display-none {
+	display: none !important;
+}
+
+.tooltip {
+	position: absolute;
+	top: 50%;
+	right: 100%;
+	transform: translateY(-50%);
+	background-color: rgba(0, 0, 0, 0.9);
+	color: #ffffff;
+	padding: 12px 15px;
+	border-radius: 6px;
+	white-space: normal;
+	opacity: 0;
+	transition: opacity 0.3s, visibility 0.3s;
+	box-shadow: 0 3px 10px rgb(0, 0, 0);
+	z-index: 1;
+	border: 1px solid #000000;
+	max-width: 350px;
+	width: max-content;
+	margin-right: 10px;
+}
+
+.tooltip::after {
+	content: "";
+	position: absolute;
+	top: 50%;
+	left: 100%;
+	margin-top: -6px;
+	border-width: 6px;
+	border-style: solid;
+	border-color: transparent transparent transparent rgba(0, 0, 0, 0.9);
+}
+
+.page-approval-container:hover .tooltip {
+	visibility: visible;
+	opacity: 1;
+}
+
+.approval-dropdown {
+	position: absolute;
+	top: 100%;
+	right: 0;
+	background-color: white;
+	box-shadow: 0 2px 5px rgba(0, 0, 0, 0.2);
+	border-radius: 4px;
+	overflow: hidden;
+}
+
+.dropdown-item {
```

Comment:
Unrelated to this PR, but we need to be careful with custom classes that match Bootstrap classes. The Chameleon skin uses Bootstrap and could have unexpected side-effects if we redeclare Bootstrap classes.

Reply:
@waximabbax

Reply:
I am aware of this but I didn't know `.dropdown-item` was a bootstrap class.